lib/ddtrace/contrib/rails/utils.rb

```diff
@@ -72,6 +72,18 @@ def self.connection_config
           end
         end
 
+        def self.exception_is_error?(exception)
+          if defined?(::ActionDispatch::ExceptionWrapper)
+            # Gets the equivalent status code for the exception (not all are 5XX)
+            # You can add custom errors via `config.action_dispatch.rescue_responses`
+            status = ::ActionDispatch::ExceptionWrapper.status_code_for_exception(exception.class.name)
```

Comment:
I guess this compatibility has been tested in the other PR, right?

Reply:
Yes it has. And it's kind of a copy-paste from the Rails code we had before.